src/scf-release/jobs/acceptance-tests-brain/templates/run.erb

```diff
@@ -48,7 +48,13 @@ for i in "${ASSETS}/plugins/"*; do
 done
 shopt -u nullglob
 
-PARAM="--timeout 600"
+PARAM=""
+
+if [ -n "${TIMEOUT}" ]; then
+    PARAM="${PARAM} --timeout ${TIMEOUT}"
+else
+    PARAM="${PARAM} --timeout 600"
```

Comment:
This can be simplified to `PARAM="${PARAM} --timeout ${TIMEOUT:-600}"`

Reply:
Hm ... Actually even `PARAM="--timeout ${TIMEOUT:-600}"` because `PARAM` is empty at that point.
Ok, will do.

Reply:
Done with commit 0be90f54d274628d6b42dd7b93a659b58a5d5865